examples/Bunch/include/simulation_defines/param/fieldBackground.param

```diff
@@ -81,7 +81,7 @@ namespace picongpu
             /* example 1: TWTS background pulse */
             /** unit: meter */
             const double WAVE_LENGTH_SI = 0.8e-6;
-            
+
             /** UNITCONV */
             /* const double UNITCONV_Intens_to_A0 = SI::ELECTRON_CHARGE_SI
                 * SI::ELECTRON_CHARGE_SI * 2.0 * WAVE_LENGTH_SI * WAVE_LENGTH_SI / (4.0 * PI * PI
```

Comment:
The reason that is not allowed to reuse the`*` is that e.g. doxygen remove the star while the documentation is generated.